cpp/src/aztec/plonk/composer/composer_base.hpp

```diff
@@ -7,8 +7,8 @@
 #include <srs/reference_string/file_reference_string.hpp>
 
 using namespace bonk;
```

Comment:
I know you didn't add this line but please delete it. This is bad form an in this case entirely unnecessary.